shared/domains/chelonia/internals.js

```diff
@@ -741,7 +742,7 @@ export default (sbp('sbp/selectors/register', {
       },
       [SPMessage.OP_CONTRACT] (v: SPOpContract) {
         state._vm.type = v.type
-        const keys = keysToMap(v.keys, height)
+        const keys = keysToMap.call(self, v.keys, height)
```

Comment:
o4-mini (high) is unhappy that in some paces in this file `self` and `this`, though the same, are used inconsistently with `.call`. It suggests picking one and sticking with it, e.g. here:

```js
--- a/shared/domains/chelonia/internals.js
+++ b/shared/domains/chelonia/internals.js
@@ -250,7 +250,7 @@ const handleEvent = {
       const stateForValidation = opT === SPMessage.OP_CONTRACT &&
         !state?._vm?.authorizedKeys
         ? {
-            _vm: { authorizedKeys: keysToMap.call(this, ((opV): any).keys, height) }
+            _vm: { authorizedKeys: keysToMap.call(self, ((opV): any).keys, height) }
           }
         : state
```